src/Platform/OpenGL/Shaders/Library/common_utils.glsl

```diff
@@ -3,6 +3,9 @@ struct Camera
     vec3 position;
     mat4 viewProjMatrix;
     mat4 invViewProjMatrix;
+    mat4 viewMatrix;
```

Comment:
There are lots of warnings 'ignore non existing uniform' as not all shaders use these matrices.
Thats my poor design of validating shaders, but this need to be fixed in the PR.
Create a method "ignore camera uniforms" with all these parameters and call it in each shader using camera. This will make all them optional

Reply:
Actually we have it. https://github.com/asc-community/MxEngine/blob/599eb5f4e55c5039a04ff5fe548bfc8d43a3d58a/src/Platform/OpenGL/ShaderBase.cpp#L360 I just not notice the warnings....